### PR TITLE
Fix JS coverage generation in crawler

### LINKED ISSUE #98: Fix the generation of JS part of the reports
See https://github.com/marco-c/coverage-crawler/issues/92#issuecomment-402325103 and following comments.

### PR DESCRIPTION
**Pull request by rhcu**
Fixes #98 
Passing a directory with `.info` file didn't work but passing a zipped file did.
I'll fill a bug in Bugzilla about `lcov_rewriter.py` and will try to fix it shortly.

### CODE REVIEW

**Comment by marco-c** [comment]
It works for me if I have an info file inside a directory.
`grcov directory/file.info` doesn't work.
`grcov directory` works.

**Comment by rhcu** [comment]
Oh, ok. Sorry for that. I've passed `directory/file.info`. I'll update it.

File: crawler.py [hunk 1]
```diff
@@ -265,12 +266,15 @@ def run_all():
                 rewriter = LcovFileRewriter(os.path.join('tools', 'chrome-map.json'))
                 rewriter.rewrite_files(jsvm_files, jsvm_output_file, '')
 
+                os.makedirs('jsvm_output', exist_ok=True)
+                shutil.move(jsvm_output_file, os.path.join('jsvm_output', jsvm_output_file))
```
Comment (marco-c) [inline crawler.py]: Instead of moving the file, you could create it directly at the right place

File: crawler.py [hunk 2]
```diff
@@ -260,7 +261,8 @@ def run_all():
 
                 from lcov_rewriter import LcovFileRewriter
 
-                jsvm_output_file = 'jsvm_lcov_output.info'
+                os.makedirs('jsvm_output', exist_ok=True)
```
Comment (marco-c) [inline crawler.py]: One last suggestion, if you put this directory inside jsvm_dir, you won't have to manually remove it.
```
jsvm_output_dir = os.path.join(jsvm_dir, 'jsvm_output')
jsvm_output_file = os.path.join(jsvm_output_dir, 'jsvm_lcov_output.info')
```

**Pull request merged**